src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAddressChange.OSX.cs

```diff
@@ -181,10 +216,17 @@ private static void StopRunLoop()
 
         private static void OnAddressChanged(IntPtr store, IntPtr changedKeys, IntPtr info)
         {
-            NetworkAddressChangedEventHandler handler = s_addressChangedSubscribers;
-            if (handler != null)
+            NetworkAddressChangedEventHandler addressChangedHandler = s_addressChangedSubscribers;
+            if (addressChangedHandler != null)
+            {
+                addressChangedHandler(null, EventArgs.Empty);
+            }
```

Comment:
Nit: with C# 6, this store-into-a-local-then-check-for-null-then-invoke can be simplified to just:
```C#
s_addressChangedSubscribers?.Invoke(null, EventArgs.Empty);
```